src/components/annotate/annotate.ts

```diff
@@ -191,8 +163,8 @@ export class AnnotateComponent extends ChromeProvider(LitElement) implements Wit
     // math so that we only have to do one calculation
     //
     // TODO: we might want to make this inclusive e.g. >=
-    const isTimeInView = model.startOffset < temporalDomain[1] && model.endOffset > temporalDomain[0];
-    const isFrequencyInView = model.lowFrequency < frequencyDomain[1] && model.highFrequency > frequencyDomain[0];
+    const isTimeInView = model.startOffset > temporalDomain[0] && model.endOffset >= temporalDomain[1];
```

Comment:
you've mixed up the domain indices here

Reply:
Thanks for catching that. Updated code is below

![image](https://github.com/user-attachments/assets/b59486cd-1618-47c4-af2d-2db0f0ece770)